packages/components/addon/components/hds/tabs/tab.hbs

```diff
@@ -0,0 +1,19 @@
+{{! template-lint-disable require-context-role }}
+{{! template-lint-disable no-invalid-role }}
+
+<li class={{this.classNames}} role="presentation">
+  <a
+    role="tab"
+    class="hds-typography-body-200 hds-font-weight-medium"
+    href={{concat "#" this.panelId}}
+    id={{this.tabId}}
+    aria-selected={{this.isSelected}}
+    tabindex={{unless this.isSelected "-1"}}
```

Comment:
is it me, or this is always `-1` no matter what? (I don't understand why...)
<img width="412" alt="modified" src="https://user-images.githubusercontent.com/686239/191610002-3470f42e-8f89-46ef-a83f-3acc01cd9fbc.png">
same for the `aria-selected`, is always "false"